tasks/dependencies.rb

```diff
@@ -78,6 +78,16 @@
     bundle "install", cwd: "acceptance", delete_gemfile_lock: !conservative
   end
 
+  def latest_gem_version(gem_name)
```

Comment:
I thought you had switched to using the api for this over shelling out but maybe I am misremembering.